### PR TITLE
Fix regular expression bit

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md [hunk 1]
```diff
@@ -70,12 +70,14 @@ The Microsoft address local part must follow these parameters:
...
+Note that the validation test checks if the local part, preceding the "+", matches the regular expression.
 
 ### All other domains
 
-For all other domains, Braze allows email addresses matching the following regex for the local part:<br>
-`/\A(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])(?:(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~\.]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])*(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}]))?\z/`
+For all other domains, Braze allows email addresses matching the following regular expression for the local part:<br>
+```
+/\A(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])(?:(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~\.]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])*(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}]))?\z/
+```
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md:80]: Unsure why there is a line break tag here 🤔 

```suggestion
For all other domains, Braze allows email addresses matching the following regular expression for the local part:

```
/\A(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])(?:(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~\.]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])*(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}]))?\z/
```
```

Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md:80]: Oops my suggestion syntax wacked this out 😭

Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md:80]: good call! I didn't notice that sneaky line break

File: _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md [hunk 2]
```diff
@@ -70,12 +70,14 @@ The Microsoft address local part must follow these parameters:
...
 - Cannot end with a period (.)
 
-Note that the validation test checks if the local part, preceding the "+", matches the regex.
+Note that the validation test checks if the local part, preceding the "+", matches the regular expression.
 
 ### All other domains
 
-For all other domains, Braze allows email addresses matching the following regex for the local part:<br>
-`/\A(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])(?:(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~\.]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}])*(?-mix:[a-zA-Z0-9_\-\^+$'\&#\/!%\*=\?`\|~]|[[^\p{ASCII}\p{Space}]&&\p{Alnum}\p{Punct}\p{S}]))?\z/`
+For all other domains, Braze allows email addresses matching the following regular expression for the local part:<br>
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/email_setup/email_validation.md:77]:
```suggestion
For all other domains, Braze allows email addresses matching the following regular expression for the local part:
```

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**